Tools/ardupilotwaf/boards.py

```diff
@@ -352,18 +365,24 @@ def build(self, bld):
 
 class px4_v1(px4):
     name = 'px4-v1'
+    toolchain = 'arm-none-eabi'
```

Comment:
See this comment: https://github.com/diydrones/ardupilot/pull/3765/commits/5e85b58f1c534dba7f440803be5e85b8ce878da4#r56676771

With that, we should be able to declare `toolchain = 'arm-none-eabi'` only in `px4` class and we wouldn't need to set it for each one of the child classes :-)